frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -392,16 +393,28 @@ class MidCircuitMeasure(HybridOp):
 
     binder = qmeasure_p.bind
 
+    def __init__(self, *args, **kwargs):
```

Comment:
I think it would be better to declare each argument so we know what to expect

Reply:
We're not using anything from `*args, **kwargs` anymore so we don't need to, right?